docs/pages/guides/using-sentry.mdx

```diff
@@ -52,146 +62,22 @@ Once you have each of these: **organization slug**, **project name**, **DSN**, a
-
-```tsx app/_layout.tsx
-import { Slot, useNavigationContainerRef } from 'expo-router';
-import { useEffect } from 'react';
-import * as Sentry from '@sentry/react-native';
-import { isRunningInExpoGo } from 'expo';
-
-// Construct a new integration instance. This is needed to communicate between the integration and React
-const navigationIntegration = Sentry.reactNavigationIntegration({
-  enableTimeToInitialDisplay: !isRunningInExpoGo(),
-});
-
-Sentry.init({
-  dsn: 'YOUR DSN HERE',
-  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  tracesSampleRate: 1.0, // Set tracesSampleRate to 1.0 to capture 100% of transactions for tracing. Adjusting this value in production.
-  integrations: [
-    // Pass integration
-    navigationIntegration,
-  ],
-  enableNativeFramesTracking: !isRunningInExpoGo(), // Tracks slow and frozen frames in the application
-});
-
-function RootLayout() {
-  // Capture the NavigationContainer ref and register it with the integration.
-  const ref = useNavigationContainerRef();
-
-  useEffect(() => {
-    if (ref?.current) {
-      navigationIntegration.registerNavigationContainer(ref);
-    }
-  }, [ref]);
-
-  return <Slot />;
-}
-
-// Wrap the Root Layout route component with `Sentry.wrap` to capture gesture info and profiling data.
-export default Sentry.wrap(RootLayout);
-```
-
-</Tab>
+### Use the Sentry wizard to set up your project
 
-<Tab label="Not using Expo Router">
+The easiest way to set up Sentry in your Expo project is to use the Sentry wizard. This tool will automatically configure your project with the right settings.
 
-Add the following to your app's main file such as **App.js**:
+Run the following command in your project directory:
 
-```js
-import * as Sentry from '@sentry/react-native';
+<Terminal cmd={['$ npx @sentry/wizard@latest -i reactNative']} />
 
-Sentry.init({
-  dsn: 'YOUR DSN HERE',
-  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-});
-```
-
-Now wrap the root component of your app with Sentry. This may be **App.js** or **index.js** depending on your project.
+The wizard will:
 
-```js
-import * as Sentry from '@sentry/react-native';
+1. Install the required dependencies
+2. Configure your project to use Sentry
+3. Set up the Metro configuration automatically
+4. Add the necessary initialization code to your app
 
-// Your App component here
-
-export default Sentry.wrap(App);
-```
-
-</Tab>
-
-</Tabs>
+Follow the prompts in the wizard to complete the setup process. The wizard will ask for your **organization slug**, **project name**, and **auth token** that you noted earlier.
```

Comment:
```suggestion
Follow the prompts in the wizard to complete the setup process. The wizard will ask for your **organization slug**, **project name**, and **auth token** that you noted earlier.
```

If I've a Sentry account, I do not need to add the details like Organization Slug, project name, and auth token on my own and instead the wizard from the CLI just asks me to login to my Sentry account. I was wondering if this could be reflected here as well and also make Step 1 shorter, so that the user who has created a new account or already has an account does not need to provide these details.

Reply:
updated in my review to be more direct about what the wizard will do